### ISSUE TITLE
dataReceived() doc string should mention bytes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8763]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8763|
|Type|enhancement|
|Created|2016-08-16 05:36:46Z|
|Branch|https://github.com/twisted/twisted/tree/8763-rodrigc-datareceived-doc|

twisted.internet.interfaces.IProtocol.dataReceived() should mention that it takes a parameter of type bytes

<details><summary>Searchable metadata</summary>

```
trac-id__8763 8763
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8763_rodrigc_datareceived_doc 8763-rodrigc-datareceived-doc
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1471325806677710 1471325806677710
changetime__1471727288815496 1471727288815496
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/502

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc set owner to @rodrigc|
|-|-|

Reviewed by itamar: https://github.com/twisted/twisted/pull/502#issuecomment-241219408

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc set status to `closed`|
|-|-|

In changeset ef662dcebf8e550c930440bae52a7d1ef9456a68
```
#!CommitTicketReference repository="" revision="ef662dcebf8e550c930440bae52a7d1ef9456a68"
Merge pull request #502 from twisted/8763-rodrigc-datareceived-doc

Author: rodrigc
Reviewer: itamar
Fixes: #8763

Clarify docstring for twisted.internet.interfaces.IProtocol.dataReceived()
```
